Doc/library/pathlib.rst

```diff
@@ -539,6 +539,20 @@ Pure paths provide the following methods and properties:
       ValueError: '/etc/passwd' does not start with '/usr'
 
 
+.. method:: PurePath.is_relative_to(*other)
+
+   Return the boolean result that this path is relative to *other* path.
```

Comment:
I agree, this would be an improvement over my earlier suggestion. Since it's modifying a verb, "Return" in this case, I think it might be slightly more correct to explicitly specify the "or not":

``Return whether or not this path is relative to the *other* path``.

Reply:
ops， i am not a native english speaker, so i have no idea about "whether(or not)" :(

Reply:
@shihai1991 That's okay, the distinction is often a pain, even for those of us who are native speakers (: